ecs-cli/modules/utils/utils.go

```diff
@@ -106,3 +107,40 @@ func GetPartition(region string) string {
 		return "aws"
 	}
 }
+
+// ParseLoadBalancers returns a list of load balacners struct
+// For users wanting to register multiple or one load balancers to a single service
+func ParseLoadBalancers(flagValue []string, list []*ecs.LoadBalancer) ([]*ecs.LoadBalancer, error) {
+
+	for i := 0; i < len(flagValue); i++ {
+		m := make(map[string]string)
+		targetGroupArn := false
```

Comment:
maybe "elbv2" as var name because this is supposed to determine if this is registering to classic LB or ALB/NLB